node/metrics/useQuote.ts

```diff
@@ -0,0 +1,63 @@
+import type { Metric } from './metrics'
+import { sendMetric } from './metrics'
+
+type UseQuoteFieldsMetric = {
+  quote_id: string
+  quote_reference_name: string
+  order_form_id: string
+  quote_creation_date: string
+  quote_use_date: string
+  creator_email: string
+  user_email: string
+  cost_center_id: string
+  buy_org_id: string
+  quote_last_update: string
+}
+
+type UseQuoteMetric = Metric & { fields: UseQuoteFieldsMetric }
+
+export type UseQuoteMetricsParams = {
+  quote: Quote
+  orderFormId: string
+  account: string
+  userEmail: string
+}
+
+const buildUseQuoteMetric = async (
```

Comment:
There's no async operation here, this function doesn't need to be async or awaited.